sys/crypto/chacha.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin, Computer Systems & Telematics
```

Comment:
no and I am not interrested in the copyright at all. do I need to add one?
(when I did some code based on older or already written code - i ll just leave it yes?)